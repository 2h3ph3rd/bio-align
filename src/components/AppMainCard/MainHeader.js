import * as React from 'react'
import Box from '@mui/material/Box'

export default function MainHeader() {
  return (
    <Box sx={style.container}>
      <img src={process.env.PUBLIC_URL + '/header.png'} alt="header" />
    </Box>
  )
}

const style = {
  container: {
    textAlign: 'center',
    mb: 1,
  },
}
