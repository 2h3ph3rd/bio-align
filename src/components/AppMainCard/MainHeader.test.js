import React from 'react'
import renderer from 'react-test-renderer'

import MainHeader from './MainHeader'

it('renders correctly', () => {
  const tree = renderer.create(<MainHeader></MainHeader>).toJSON()
  expect(tree).toMatchSnapshot()
})
