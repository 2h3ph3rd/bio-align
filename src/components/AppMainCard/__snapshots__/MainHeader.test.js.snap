// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiBox-root css-co4xce"
>
  <img
    alt="header"
    src="/header.png"
  />
</div>
`;
